.github/workflows/test.yml

```
@@ -15,6 +15,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+    - name: Setup python
```

```suggestion
    - name: use python v3.9
```

nit, for consistency.